server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMessagesUpdateProcessorTest.java

```diff
@@ -66,7 +66,7 @@ public void processShouldReturnNonEmptyNotUpdatedWhenRequestHasInvalidUpdate() {
         when(mockConverter.fromJsonNode(any(ObjectNode.class)))
                 .thenReturn(mockInvalidPatch);
 
-        SetMessagesUpdateProcessor sut = new SetMessagesUpdateProcessor(mockConverter, null);
+        SetMessagesUpdateProcessor sut = new SetMessagesUpdateProcessor(mockConverter, null, null, null, null);
```

Comment:
Yes, it is. We use SetMessagesUpdateProcessor with one more SystemMailboxesProvider parameter constructor